<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditFilePath"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>No audio file loaded</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelJSON">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Parameters(JSON)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditJSON">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufunction">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>operation</string>
    </property>
    <addaction name="actionoriginal"/>
    <addaction name="actionreverse"/>
    <addaction name="actionfaded"/>
    <addaction name="actionvolume"/>
    <addaction name="actionresampling"/>
    <addaction name="actionOLA"/>
    <addaction name="actionDTW"/>
   </widget>
   <addaction name="menufunction"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionreverse">
   <property name="text">
    <string>reverse</string>
   </property>
  </action>
  <action name="actionfaded">
   <property name="text">
    <string>faded</string>
   </property>
  </action>
  <action name="actionvolume">
   <property name="text">
    <string>volume</string>
   </property>
  </action>
  <action name="actionoriginal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>original</string>
   </property>
  </action>
  <action name="actionresampling">
   <property name="text">
    <string>resampling</string>
   </property>
  </action>
  <action name="actionOLA">
   <property name="text">
    <string>OLA</string>
   </property>
  </action>
  <action name="actionDTW">
   <property name="text">
    <string>DTW</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
